S2VX.Game.Tests/HeadlessTests/EditorNoteTests.cs

```diff
@@ -1,16 +1,26 @@
 using NUnit.Framework;
 using osu.Framework.Allocation;
+using osu.Framework.Audio;
+using osu.Framework.Screens;
 using osu.Framework.Testing;
 using osu.Framework.Timing;
+using S2VX.Game.Editor;
 using S2VX.Game.Story;
 using S2VX.Game.Story.Command;
 using S2VX.Game.Story.Note;
+using System.IO;
 
 namespace S2VX.Game.Tests.HeadlessTests {
     [HeadlessTest]
     public class EditorNoteTests : S2VXTestScene {
         [Cached]
         private S2VXStory Story { get; set; } = new S2VXStory();
+#pragma warning disable IDE0052 // Remove unread private members
+        private EditorScreen Editor { get; set; }
+#pragma warning restore IDE0052 // Remove unread private members
```

Comment:
Don't suppress this. Just remove the variable if it's not being used.

Reply:
A reference to Editor is needed in this file since EditorNotes is now dependent on it. If I remove it the tests will break from dependency error. I was thinking of removing the StoryClock from the tests and using the Editor to seek so it's used but I don't know if there's an Editor equivalent of StoryClock.Start() so I just suppressed it for both tests

Reply:
There's Editor.Seek()

Reply:
See my other comment. You can add the Editor without assigning it.

Reply:
Fixed